docs/r3/beyondcandc.md

```diff
@@ -32,16 +33,26 @@ A well-known centralised mechanism for reliable synchronisation of state changes
 
 Each network has the ability to carry out an on-network transaction placing an asset in a "prepared" state from which a subsequent on-network transaction can either return it to its prior state (a "roll back"), or move it forward into its intended final state (a "commit").
```

Comment:
I',m not sure I love "roll back", that has a very specific meaning and even though we're undoing a change it's still a roll forward

Reply:
Modified to "revert"